distributed/protocol.py

```diff
@@ -63,6 +63,14 @@
     default_compression = 'lz4'
 
 
+with ignoring(ImportError):
+    import blosc
+    blosc.set_nthreads(2)
```

Comment:
Have you tried to play with increasing this to see its impact on compression speed?